lib/google_sign_in_desktop.dart

```diff
@@ -456,6 +460,91 @@ class GoogleSignInDesktop extends GoogleSignInPlatform {
       await server.close();
     }
   }
+
+  @override
+  // ignore: avoid_renaming_method_parameters
+  Future<bool> requestScopes(List<String> requestedScopes) async {
```

Comment:
Could we avoid renaming the method parameter?

```suggestion
  @override
  Future<bool> requestScopes(
    List<String> scopes,
  ) async {
```